tests/testStreamOpen.cpp

```diff
@@ -526,3 +526,13 @@ TEST_F(StreamOpenInput, OpenSLESInputSetPrivacySensitiveModeGetsUnspecified){
     ASSERT_EQ(mStream->getPrivacySensitiveMode(), PrivacySensitiveMode::Unspecified);
     ASSERT_TRUE(closeStream());
 }
+
+TEST_F(StreamOpenInput, OldAndroidVersionInputSetPrivacySensitiveModeGetsUnspecified){
+    if (getSdkVersion() < __ANDROID_API_R__) {
+        mBuilder.setDirection(Direction::Input);
+        mBuilder.setInputPreset(InputPreset::VoiceCommunication);
```

Comment:
If you try to set privacy when <R does it crash?

Reply:
No, it does not crash. It simply ignores the parameter.

Reply:
Oops. Nice catch!